### PR TITLE
Add operations handlers for helm release and feature resource

### PR DESCRIPTION
**Pull request by Shubbhang351**
1. **What this PR does / why we need it**:

Add operations handlers for helm release and feature resource

### CODE REVIEW

**Comment by ishangupta-ds** [comment]
@Shubbhang351 please update this PR with the changes that we discussed.

File: internal/resources/helmrelease/spec/spec.go [hunk 1]
```diff
@@ -237,3 +257,36 @@ func HasSpecChanged(d *schema.ResourceData) bool {
...
+
+func fileExists(filepath string) bool {
+	fileinfo, err := os.Stat(filepath)
+
+	if os.IsNotExist(err) {
+		return false
+	}
+
+	return !fileinfo.IsDir()
+}
```
Comment (ishangupta-ds) [inline internal/resources/helmrelease/spec/spec.go:292]: Can you log the errors? Please log at other places too.

**Review by ishangupta-ds — CHANGES_REQUESTED** [review]
The rest looks good.

**Comment by ramya-bangera** [comment]
@Shubbhang351 - Check your commit history, I think there is duplicate commits for models and clients

**Review by ramya-bangera — COMMENTED** [review]
LGTM

File: internal/resources/helmrelease/spec/spec.go (hunk 1 above)
Comment (Shubbhang351) [reply internal/resources/helmrelease/spec/spec.go]: Updated PTAL

**Review by ishangupta-ds — APPROVED** [review]
LGTM!

**Comment by ishangupta-ds** [comment]
Please rebase before merge.

**Comment by Shubbhang351** [comment]
> @Shubbhang351 - Check your commit history, I think there is duplicate commits for models and clients

before merging I'll directly merge on feature branch.

**Review by ramya-bangera — APPROVED** [review]

**Pull request merged**